examples/developer-hub-javascript/fassetsSwapAndRedeem.ts

```diff
@@ -0,0 +1,71 @@
+// 1. Dependancies and constants
+import { run } from "hardhat";
+
+import { SwapAndRedeemInstance } from "../../typechain-types";
+import { ERC20Instance } from "../../typechain-types/@openzeppelin/contracts/token/ERC20/ERC20";
+
+// AssetManager address on Songbird Testnet Coston network
+const ASSET_MANAGER_ADDRESS = "0x56728e46908fB6FcC5BCD2cc0c0F9BB91C3e4D34";
```

Comment:
We should add this to contract registry

Reply:
I will add a task to do it. On Coston there are multiple versions of asset manager and need to ask Iztok and validate which ones are working with sample code.